contrib/systemd/nv-dhcp.socket

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Netavark DHCP Proxy for per-connection netavark clients
+
+[Socket]
+ListenStream=%t/nv-dhcp/nv-dhcp-proxy.sock
+Accept=yes
```

Comment:
`Accept=yes` is probably not needed. As the name of .socket and .service matches.